website/docs/components/flags/examples/aria-label.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Flags, { iso2Name } from 'intergalactic/flags';
+import { Text } from 'intergalactic/typography';
+import { Flex } from 'intergalactic/flex-box';
+
+const countries = ['US', 'DE', 'ES', 'FR', 'IT'];
+
+const Demo = () => (
+  <>
+    {countries.map((country) => (
+      <Flags
+        key={country}
+        name={country as keyof typeof iso2Name}
+        role='image'
+        aria-label={iso2Name[country]}
+        mr={1}
```

Comment:
I would suggest to add mb={3} here just to visually separate first row with the following column a little bit. Something like this:
<img width="716" alt="image" src="https://github.com/semrush/intergalactic/assets/77798437/737f3f5e-6077-4447-aa7d-83249c00285d">

Reply:
Great example btw!

Reply:
Added the margin, thanks!